slices/libgcc-s1.yaml

```diff
@@ -3,6 +3,8 @@ package: libgcc-s1
 slices:
   libs:
     essential:
+      - gcc-14-base_copyright
```

Comment:
Maybe add a `libgcc-s1_copyright` slice which contains the `/usr/share/doc/libgcc-s1` path below. Then, list that (instead of `gcc-14-base_copyright`) as an essential of `libs`.

Reply:
I don't think there should be a separate `libgcc-s1_copyright` slice: it's important that the copyright file for any slice always be included, in order to understanding the licensing terms of any software component included in a chiseled container.

Reply:
>  it's important that the copyright file for any slice always be included, in order to understanding the licensing terms of any software component included in a chiseled container.

I agree. What I am asking is that have a separate slice for copyright files and add that as an essential of all the slices. So for example, if tomorrow, we feel the need to add a new `lintian` slice containing the lintian-overrides, we can simply do the following without re-writing the copyright path:

```yaml
libs:
  essential:
    - libgcc-s1_copyright
    - ...
  ...
lintian:
  essential:
    - libgcc-s1_copyright
    - ...
```

Reply:
Ahh, I see what you mean now, sorry!
Yes, I'll update these PR's to do that.